drift/lib/src/runtime/manager/manager.dart

```diff
@@ -389,8 +391,8 @@ abstract class BaseTableManager<
   ///
   /// Uses the distinct flag to ensure that only distinct rows are returned
```

Comment:
Let's update the comment as well, e.g. with

```suggestion
  /// The [distinct] parameter (enabled by default) controls whether to generate
  /// a `SELECT DISTINCT` query, removing duplicates from the result.
```

Reply:
Done